src/instantsend/signing.h

```diff
@@ -25,12 +25,23 @@ class CQuorumManager;
 } // namespace llmq
 
 namespace instantsend {
-class InstantSendSigner : public llmq::CRecoveredSigsListener
+class InstantSendSignerParent
```

Comment:
`InstantSendSignerParent` is meant to be used outside of `instantsend::` but  `CInstantSendManager` to be hidden inside; may be even rename `CInstantSendManager` to `InstantSendImplementation`.

please, rename `InstantSendSignerParent` to `InstantSendInterface` or `InstantSendManager`.